### ISSUE TITLE
Feature Request: Face tranforms (boxes + keypoints)

### ISSUE THREAD

**Issue by ternaus** [albumentations-team/albumentations#667]
It would be nice to have a transform that can deal with faces.

The face is a set of keypoints or bounding box with a set of keypoints.

As input, it should 

* take N faces and N*K keypoints. Each face has K keypoints.
* symmetric mapping for the transform. I.e. what key points should swap within each face. For example, left eye and right eye.

We want:

 ### Symmetrics flips
 When we flip the image, bounding boxes flip, keypoints flip + symmetric key points within each face swap.

### Crops

* When we crop part of the image and bounding box goes away - all key points in the box should go away.
* When part of the box is cropped, key points that got cut should be clipped to the side of the image.

**Comment by Dipet** [comment]
> When part of the box is cropped, key points that got cut should be clipped to the side of the image.

Are you sure that this behavior is correct?
For me this behavior is strange on example below.

![image](https://user-images.githubusercontent.com/7512250/88681928-c1cc1400-d0fa-11ea-9766-1df2ba4ce5ac.png)

**Comment by ternaus** [comment]
Yep. But I expect the behavior of the box being cropped to happen at the side of the image which is fine.

In the middle of the image, it looks strange indeed.

**Comment by cannguyen275** [comment]
Any updates for this?
This library seems to augment a set of keypoints and bounding boxes separately! But having many tasks both keypoints and bounding boxes have to be augmented together for any single object, etc. [RetinaFace](https://arxiv.org/pdf/1905.00641.pdf), [MTCNN](https://arxiv.org/pdf/1604.02878.pdf)
The below image shows a sample of landmark and bounding box have to go along with each face:
![sample](https://user-images.githubusercontent.com/35292523/92672315-9f0c5c80-f342-11ea-8059-20ac882d5f60.jpg)

**Comment by Dipet** [comment]
Hi. We are still thinking about how to implement this functionality using library interface.
